docs/how_to_run.md

```diff
@@ -0,0 +1,187 @@
+* **[OPTIONAL]** An email is sent reporting the counts for each submission status. The email includes a CSV file with the batch records from DynamoDB, reflecting the latest information.
+
+### Run DSS
+DSS consumes the submission messages from the input queue in SQS. DSS uses a client to interact with DSpace. For each item submission, DSS reads the metadata JSON file and bitstreams from S3, using the information provided in the message, and creates an item with bitstreams in DSpace.
+
+At the end of this step:
+* Result messages are written to the output queue for DSC (`dss-output-dsc`).
+
+  Note: The message is structured in accordance with the [Result Message Specification](https://github.com/MITLibraries/dspace-submission-service/blob/main/docs/specifications/result-message-specification.md).
+
+### Inspect ingest results
+DSC consumes result messages from its output queue, parsing the messages to determine whether the associated item was ingested into DSpace. It then loops through the batch records from DynamoDB, updating those that have a corresponding result message. Additional steps with the item submission may be performed on behalf of the requestor (e.g., custom reports).
+
+At the end of this step:
+* Batch records in DynamoDB are updated. Updates are made to the folllowing fields:
+    - `dspace_handle`: Set to generated DSpace handle (if item was ingested)
+    - `status`: Indicates ingest status
+    - `status_details`: Set to error messages (if item failed ingest)
+    - `last_result_message`: Set to result message string
+    - `last_run_date`: Set to current run date
+    - `ingest_attempts`: Increments by 1 
+- Result messages are deleted from the queue
+    - If any errors occur during the processing of result message, the result message will remain in the queue.
+- An email is sent reporting the counts for each ingest status. The email includes a CSV file with the batch records from DynamoDB, reflecting the latest information.
+
+## The DSC CLI
+
+### `pipenv run dsc`
+
+```text
+Usage: -c [OPTIONS] COMMAND [ARGS]...
+
+  DSC CLI.
+
+Options:
+  -w, --workflow-name TEXT  The workflow to use for the batch of DSpace
+                            submissions  [required]
+  -b, --batch-id TEXT       A unique identifier for the workflow run, also
+                            used as an S3 prefix for workflow run files
+                            [required]
+  -v, --verbose             Pass to log at debug level instead of info
+  --help                    Show this message and exit.
+
+Commands:
+  create     Create a batch of item submissions.
+  finalize   Process the result messages from the DSS output queue...
+  reconcile  Reconcile bitstreams with item identifiers from the metadata.
+  submit     Send a batch of item submissions to the DSpace Submission...
+  sync       Sync data between two directories using the aws s3 sync...
+```
+
+### `pipenv run dsc -w <workflow-name> -b <batch-id> create`
```

Comment:
You will notice that when using DSC, we will need to provide the `--workflow-name / -w` and `--batch-id / -b` (CLI options even if just want to run `<command-name> help`. This is because we currently define the options on the `main` command group and set them as `required=True`. 

We could either:

A. Decorate each command with the `-w` and `-b` CLI options and remove setting in `main` command group.
B. Create a `shared_cli_options` decorator that contains all CLI options used by the CLI commands (i.e., see [example in CDPS-CURT](https://github.com/MITLibraries/cdps-curt/blob/main/curt/cli.py#L18-L59))
C. Do nothing 

@ghukill @ehanson8 Let me know what you think! This would, of course, be outside the scope of this PR.